llama_stack/providers/utils/telemetry/trace_protocol.py

```diff
@@ -19,17 +17,17 @@
 def serialize_value(value: Any) -> Any:
     """Serialize a single value into JSON-compatible format."""
     if value is None:
-        return None
+        return ""
     elif isinstance(value, (str, int, float, bool)):
         return value
+    elif hasattr(value, "_name_"):
```

Comment:
what's this?!

Reply:
for handling enums.

Reply:
@dineshyv you should check for enums explicitly 

```
elif instance(v, Enum): 
   return v.value
```

?